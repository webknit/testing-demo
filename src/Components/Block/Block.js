
function Block({ title }) {
    return (
        <div style={{ backgroundColor: 'blue', padding: '2rem' }}>
            <h1>{title}</h1>
        </div>
    );
}

export default Block;
