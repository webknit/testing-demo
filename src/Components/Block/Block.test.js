
import renderer from 'react-test-renderer';
import Block from './Block';

// import data from './__mocks__/carCarouselData.json';

describe('<Block />', () => {
    describe('snapshot tests', () => {
        test('should render with no data', () => {

            const tree = renderer
                .create(<Block title='hello' />)
                .toJSON();
            expect(tree).toMatchSnapshot();
        });

        // test('should render with data', () => {
        //     const { snapshot } = generateSnapshot({ Component: CarCarousel, mockData: data, inApp: false });
        //     expect(snapshot).toMatchSnapshot();
        // });
    });

});