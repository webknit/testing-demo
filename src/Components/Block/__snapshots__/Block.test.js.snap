// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Block /> snapshot tests should render with no data 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "padding": "2rem",
    }
  }
>
  <h1>
    hello
  </h1>
</div>
`;
